### PR TITLE
Fix sorting bug for tags caused by changes in API

### PR DESCRIPTION
**Pull request by niklasva82**
Fixes sorting bug caused by new tag view column which includes values for value tags.

To reproduce bug:
1. Select a view
2. Add a tag column
3. Assign a tag to one of the people in the view
4. Click the sort button in the header of the tag column

Expected:
Column should sort on the presence of the specified tag.

Actual:
Nothing happens.

### CODE REVIEW

File: src/components/misc/personViews/PersonViewTable.jsx [hunk 1]
```diff
@@ -162,7 +165,7 @@ export default class PersonViewTable extends React.Component {
                                 const joinedOptions0 = latest0.selected.reduce(joinText, '');
                                 const joinedOptions1 = latest1.selected.reduce(joinText, '');
                                 x = joinedOptions0.localeCompare(joinedOptions1);
-                            }
+                            }
```
Comment (richardolsson) [inline src/components/misc/personViews/PersonViewTable.jsx:168]: I'm not happy about little things like this, but I'm glad we're avoiding them in app.zetkin.org thanks to linting and auto-formatting. 😊

File: src/components/misc/personViews/PersonViewTable.jsx [hunk 2]
```diff
@@ -139,10 +139,13 @@ export default class PersonViewTable extends React.Component {
 
                             let x = 0;
 
-                            if (colType == 'local_bool' || colType == 'person_tag' || colType == 'person_query') {
+                            if (colType == 'local_bool' || colType == 'person_query') {
                                 // Treat boolean values as integers (1 or 0)
                                 x = +val1 - +val0;
                             }
+                            else if (colType == 'person_tag') {
+                                x = !!val0 < !!val1 ? 1 : -1;
```
Comment (richardolsson) [inline src/components/misc/personViews/PersonViewTable.jsx:147]: Nice solution!

**Review by richardolsson — APPROVED** [review]
This looks good to me!

**Comment by niklasva82** [comment]
Haha forgot that I had already solved this :smile:

**Pull request merged**